scripts/verify-replace.sh

```diff
@@ -16,27 +19,45 @@ do
     echo                                                                     
     echo =========================================================================================                                            
     repo_path=${BASE_REPO_PATH}/$(basename ${repo})
+    ERRFILE=$(mktemp ${TMP_DIR}$(basename ${repo})-error.XXX)
     echo "Cloning repo in /tmp"
     git clone --depth=1 ${repo} ${repo_path}
     echo "Repo cloned successfully"
     cd ${repo_path}
     if ! make pre-verify; then
-        ERRORLIST+="($(basename ${repo}))"
+        ERRORREPOLIST+="$(basename ${repo}) "
         continue
     fi
     echo "Initiating 'go mod replace' of current toolchain common version in dependent repos"
     go mod edit -replace github.com/codeready-toolchain/toolchain-common=${C_PATH}
-    make verify-dependencies || ERRORLIST+="($(basename ${repo}))"
+    make verify-dependencies &> ${ERRFILE} 
```

Comment:
IIRC, you mentioned that the only process that doesn't print errors to a stderr is the linter, wouldn't it be better to redirect the outputs to separate files and:
* print the whole content of the file with stderr
* and parse the stdin with the `GOLINTREGEX`?

In this way, you could limit the dependency on the error output formatting.

Reply:
Yes, I like your idea of reducing the dependency, I have changed the logic now. PTAL